compiler/src/iree/compiler/Codegen/SPIRV/test/tile_and_vectorize_to_cooperative_ops.mlir

```diff
@@ -341,3 +341,335 @@ hal.executable public @matmul_256x1024x128_div_add {
 //       CHECK:         %[[READ6:.+]] = vector.transfer_read %{{.+}}[%[[C0]], %[[C0]], %[[C0]]]
 //       CHECK:         %[[DIV:.+]] = arith.divf %[[READ6]], %[[READ5]] : vector<1x16x16xf16>
 //       CHECK:         vector.transfer_write %[[DIV]], %{{.+}}[%[[C0]], %[[C0]], %[[C0]]]
+
+// -----
```

Comment:
These tests are just duplications from the above (only changing the vendor and cooperative matrix capabilities)? Is this true? If so we don't need them. Note that this is only testing the particular `-iree-spirv-tile-and-vectorize-to-cooperative-ops` pass, with configuration already embedded; so the original target environment does not matter anymore.

Reply:
Ditto on the above, there are slight differences in the generated code, I assume mainly due to the different subgroup size, although this should already be covered by the pipeline tests so these are probably redundant.

Reply:
Yeah this does not matter as the core logic of the pass is already checked by previous tests; the exact subgroup size does not alter the CodeGen flow (but would affect generated IR for sure.) So let's remove these tests.